File: TelegramConsumer/Entities/Media.cs
using System.Text.Json.Serialization;

namespace TelegramConsumer
{
    public class Media
    {
        [JsonPropertyName("url")]
        public string Url { get; set; }

        [JsonPropertyName("type")]
        public MediaType Type { get; set; }

        [JsonPropertyName("thumbnail_url")]
        public string ThumbnailUrl { get; set; }
        
        [JsonPropertyName("duration_seconds")]
        public int DurationSeconds { get; set; }

        [JsonPropertyName("width")]
        public int Width { get; set; }

        [JsonPropertyName("height")]
        public int Height { get; set; }
    }
}

File: updatesproducer/updateapi/media.py
from updatesproducer.updateapi.mediatype import MediaType


class Media:
    url: str
    type: MediaType
    thumbnail_url: str
    duration_seconds: int
    width: int
    height: int

    def __init__(
            self,
            url: str,
            type: MediaType):
        self.url = url
        self.type = type

    def __init__(
            self,
            url: str,
            type: MediaType,
            thumbnail_url: str,
            duration_seconds: int,
            width: int,
            height: int):
        self.url = url
        self.type = type
        self.thumbnail_url = thumbnail_url
        self.duration_seconds = duration_seconds
        self.width = width
        self.height = height
